pom.xml

```diff
@@ -1,311 +1,345 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The POM had some inconsistent indentation so I reformatted it, our standard is 2 spaces instead of 4 so this diff now looks a lot bigger than it actually is. If people are very adamant about using 4 spaces I can put it back but I personally prefer 2. I'll put comments below highlighting what I actually changed.